### ISSUE TITLE
dotnet migrate does not generate <Optimize>true</Optimize> when "optimize": true is used

### ISSUE THREAD

**Issue by pdeligia** [dotnet/cli#5885]
## Steps to reproduce

Run `dotnet migrate` to migrate the following example json project file to a csproj file.
```
{
  "version": "1.0.0-*",
  "buildOptions": {
    "optimize": true,
    "allowUnsafe": true
  },
  
  "runtimes": {
    "win7-x64": { }
  },

  "frameworks": {
    "netstandardapp1.7": {
      "dependencies": {
          "Microsoft.NETCore.Platforms" : "2.0.0-beta-25101-0",
          "Microsoft.Private.CoreFx.NETCoreApp" : "4.4.0-beta-25101-0"
      }
    }
  }
}
```

## Expected  behavior

The migrated csproj file should include the `<Optimize>true</Optimize>` , since `"optimize": true` was used.

## Actual behavior

In the migrated csproj file, `<Optimize>true</Optimize>` is missing.

## Environment data
`dotnet --info` output:

.NET Command Line Tools (2.0.0-alpha-004853)

Product Information:
 Version:            2.0.0-alpha-004853
 Commit SHA-1 hash:  e7689a3116

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0-alpha-004853\

**Comment by pdeligia** [comment]
Maybe related to #5827 ?

**Comment by piotrpMSFT** [comment]
yea, likely related. The workaround here is straightforward, but let's try to get this fixed for 2.0

**Comment by livarcocc** [comment]
This issue was moved to dotnet/cli-migrate#21
